lib/streamlit/commands/execution_control.py

```diff
@@ -106,7 +107,7 @@ def switch_page(page: str) -> NoReturn:  # type: ignore[misc]
 
     Parameters
     ----------
-    page: str
+    page: str or st.Page
         The file path (relative to the main script) of the page to switch to.
```

Comment:
nit: it looks like the Parameter description needs to be updated now that page is supported. Also, perhaps the type here should be `StreamlitPage` but no strong opinion here

Reply:
Updated the docstring. I think this is a good question for docs, but `StreamlitPage` is a type for `st.Page`, but I don't think people will understand that.